setup.cfg

```
@@ -38,6 +38,7 @@ install_requires =
     mistune>=0.8.1
     nbconvert>=7,<8
     nbformat>=5.1,<6.0
+    lxml_html_clean>=0.1.1
```

```suggestion
    lxml-html-clean>=0.1.1
```

minor: to match the PyPI package https://pypi.org/project/lxml-html-clean/

We've also been bitten in the past from this, since PyPI/pip (or some other tool) automatically converts in dependency strings underscores to dashes...